draft-ietf-tls-dtls-rrc.md

```diff
@@ -179,32 +179,68 @@ layer specific address validation mechanism can be triggered instead.
 
 # Attacker Model
 
-RRC is designed to withstand the following attacks:
+We define two classes of attackers, off-path and on-path, with increasing
+capabilities (see {{fig-attacker-capabilities}}):
 
-* On-path attackers that try to trigger a connection migration ({{on-path}});
-* Off-path attackers that try to put themselves on-path ({{off-path}}) --
-  provided that the enhanced path validation algorithm is used, see
-  {{enhanced}}.
+* An off-path attacker is not on the original path between the DTLS peers, but
+  is able to observe packets on the original path and has faster routing
+  compared to the DTLS peers, which allows it to make copies of the observed
+  packets, race its copies to either peer and consistently win the race.
```

Comment:
In this case "win" is an infinitive with an elided "to" for brevity / to avoid redundancy.

It'd be "[...] to race its copies to either peer and to consistently win the race."